tox.ini

```
@@ -160,4 +160,137 @@ commands_post =
     echo "After codespell has been installed locally (`pip install codespell`),"
     echo "running the command `codespell -i 2 -w` will interactively go through"
     echo "misspellings and suggest one or more replacements. Add any false"
-    echo "positives under ignore-words-list under [codespell] in setup.cfg."
+    echo "positives under ignore-words-list under [codespell] in pyproject.toml."
+
+[flake8]
+convention = numpy
+# Checks marked as TODO should be eventually enabled,
+# but are failing right now. These are *great*
+# first contributions. ;)
+# Checks not marked by TODO *maybe* should be
+# eventually enabled ― ask!
+extend-ignore =
+    D105,
+    D107,
+    # Ignoring D202 allows blank lines to be put on either side of code "paragraphs" at the beginning of a function.
+    D202,
+    # D205 and D400 are ignored to allow the "one-liner" to exceed one
+    # line, which is sometimes necessary for even concise descriptions of plasma
+    # physics functions and classes.
+    D205,
+    # D302 is unnecessary as we are using Python 3.6+.
+    D302,
+    # D205 and D400 are ignored to allow the "one-liner" to exceed one
+    # line, which is sometimes necessary for even concise descriptions of plasma
+    # physics functions and classes.
+    D400,
+    # D403: First word of the first line should be properly capitalized ('Latex', not 'LaTeX') # Dominik: good job, pydocstyle /s
+    D403,
+    # E203 something something whitespace? TODO check again
+    E203,
+    E501,
+    # E731 do not assign a lambda, use a def - Dominik: disagreed, lambdas have uses
+    E731,
+    F401,
+    F405,
+    # W605 invalid escape sequence,
+    W605,
+    RST210,
+    RST213,
+    # RST305 so that ReST substitutions in a different file can be used
+    RST305,
+    RST306
```

```suggestion
    RST306
    # TC003 to allow long messages to be specified outside the exception class
    TC003
```
Adding in a `tryceratops` error code to ignore (from #1782).